lib/pages/my_reports_pages/my_reports_page.dart

```diff
@@ -486,7 +486,7 @@ class _MyReportsPageState extends State<MyReportsPage> {
                                             children: [
                                               Style.body(
                                                   MyLocalizations.of(
-                                                      context, 'more_info'),
+                                                      context, 'more_info') + ': ',
```

Comment:
I can't compile the project
```
The operator '+' can't be unconditionally invoked because the receiver can be 'null'.
Try adding a null check to the target ('!').dart[unchecked_use_of_nullable_value](https://dart.dev/diagnostics/unchecked_use_of_nullable_value)
String +(String other)
```

This fixes it, it has to be added in other places in this file too
```
context, 'more_info')! + ': ',
```

Reply:
Fixed on [bb85100](https://github.com/Mosquito-Alert/Mosquito-Alert-Mobile-App/pull/86/commits/bb8510037f75befda5a2ece3a1d7ffeae831fdb6)